src/artworks/artwork.constraints.ts

```diff
@@ -1,16 +1,15 @@
 export const ARTWORK_CONSTRAINTS = {
-    title: {
-        presence: true
-    },
-    imageUrl: {
-        presence: true,
-        url: true
-    },
-    artistId: {
-        presence: true,
-        personExists: true
-    },
-    eventbriteId: {
-        presence: false
-    }
+	title: {
```

Comment:
Can we change this to indentation using 4 spaces? I know this is linting issue again 🤣